mkdocs.yml

```diff
@@ -23,6 +26,8 @@ theme:
     - content.action.edit
     - content.code.copy
     - content.tooltips
+    - content.code.annotate
```

Comment:
this allows for those expandable plus signs in code blocks

Reply:
For long term maintainability, might be nice to link to the relevant sections of the Material docs and call out what these each do?